build/aten/src/ATen/TypeDefault.cpp

```diff
@@ -771,12 +771,9 @@ Tensor & TypeDefault::_baddbmm_mkl_(Tensor & self, const Tensor & batch1, const
     const OptionalDeviceGuard device_guard(device_of(self));
     return at::native::_baddbmm_mkl_(self, batch1, batch2, beta, alpha);
 }
-Tensor TypeDefault::bartlett_window(int64_t window_length, const TensorOptions & options) {
-#ifdef BUILD_NAMEDTENSOR
-
-#endif
-    const DeviceGuard device_guard(options.device());
```

Comment:
This guy should stay and look like this: `const DeviceGuard device_guard(device.value());`